doc/distrib/NodeHelpFiles/Revit.Elements.Views.View.Origin.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+Every view in Revit has an origin. `View.Origin` returns this value as a Dynamo point. According to the Revit API documentation, "The origin of a plan view is not meaningful". With this in mind, the value offered by `View.Origin` is up to the end-user.
```

Comment:
Do you mean that you don't really see value in this node? Does it make sense to have a node for this, then?

Reply:
I think it is worth having the help file as that is what the Revit API says as well

Reply:
Especially if anyone is wondering what it does.